tests/system/main_test.go

```diff
@@ -46,7 +39,7 @@ func setup() {
 func teardown() {
 	if !*dry {
 		if err := filepath.Walk("cases", deleteStateFiles); err != nil {
-			fmt.Println("Error occurred during teardown: ", err)
+			log.Println("[ERROR] Error occurred during teardown: ", err)
```

Comment:
Where did we end up landing re: using the standard logger for `[ERROR]`? Still curious why we would log this versus just print to stdout. I'm thinking mainly of how this would best work in a CI pipeline some day (for example) and what would be most appropriate.

Reply:
For messages involving the `T` object, I don't give it a log level. For things printed via `fmt` I use log levels. I'm not using `log.Errorf` for the reasons we discussed previously about "handling" error messages, opting to just print that an error has occurred and have something else actually do the graceful error handling.